backend/modules/metadata_store/mlfoundry.py

```diff
@@ -181,7 +181,7 @@ def get_collections(
     ) -> List[Collection]:
         ml_runs = self.client.search_runs(
             ml_repo=self.ml_repo_name,
-            filter_string=f"params.type = '{MLRunTypes.COLLECTION.value}'",
+            filter_string=f"params.type = '{json.dumps(MLRunTypes.COLLECTION.value)}'",
         )
```

Comment:
This would render as
`params.type = '"xyz"'`

I am not sure about the nesting of single and double quotes, just check once